Deployment/outputs.tf

```diff
@@ -19,6 +19,10 @@ output "erp_facade_web_app_url" {
   value = "https://${module.webapp_service.default_site_hostname}"
 }
 
+output "erp_facade_web_app_public_url" {
```

Comment:
This is hard coded to dev - should it use the env_name instead?

Reply:
Only ever used in dev, but yes we should construct it using the env_name